evaluate/report/csv.go

```diff
@@ -57,7 +61,43 @@ func (e *EvaluationFile) WriteEvaluationRecord(model model.Model, language langu
 	return nil
 }
 
+// WriteEvaluationRawRecords takes a slice of raw records and writes them into the evaluation file.
+func (e *EvaluationFile) WriteEvaluationRawRecords(rawRecords []string) (err error) {
```

Comment:
Would rename to `WriteLines` as this one is a generic method which simply writes string lines into the CSV file.

Reply:
Please also refactor the function `WriteEvaluationRecord` to use the new generic way of writing a line.